pkg/component/nodemanagement/machinecontrollermanager/machine_controller_manager.go

```diff
@@ -265,6 +266,7 @@ func (m *machineControllerManager) Deploy(ctx context.Context) error {
 	}
 
 	if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, m.client, vpa, func() error {
+		vpa.Labels = map[string]string{v1beta1constants.LabelExtensionWebhookControlplaneSelector: "true"}
```

Comment:
We usually pay attention to do not overwrite the whole labels/annotations to allow human operators or other components to also set a label/annotation.

```suggestion
		metav1.SetMetaDataLabel(&vpa.ObjectMeta, v1beta1constants.LabelExtensionWebhookControlplaneSelector, "true")
```